lib/experimental/interactivity-api/scripts.php

```diff
@@ -7,20 +7,21 @@
  */
 
 /**
- * Move interactive scripts to the footer. This is a temporary measure to make
- * it work with `wp_store` and it should be replaced with deferred scripts or
- * modules.
+ * Move interactive scripts to the footer and make them load with the defer strategy.
```

Comment:
do they still need to be in the footer if deferred?

Reply:
+1, arguably having them in the header would have a small benefit of allowing early discovery.

Reply:
Oh, I suppose what is meant by the original `wp_store` comment is the script being able to see the `wp_store` attributes in the HTML. I'm not familiar with the specific constraints for why it was put in the footer in the first place, so I'll _defer_ to @luisherranz. 

What I'm also curious about with the Interactivity API is whether it can load in the `head` and employ an `async` loading strategy the same way that is implemented in this PR for the Navigation block and Search block. Ideally these blocks should be interactive as soon as possible before the DOM fully loads for the reasons explained in the PR description.